### PR TITLE
Add run time to the task grid

### PR DESCRIPTION
**Pull request by ascobie**
Fix: #1501

image

Shows success and failure color matching to state column.

### CODE REVIEW

**Comment by ascobie** [comment]
If you want to complete this for release 17.0 before I get a chance to write the tests. I have a TODO to move the remaining test specs into the correct locations and will sort out the tests for task-preview and task-runtime tomorrow.

**Comment by timotheeguerin** [comment]
should you move that just right of the state?

**Comment by ascobie** [comment]
I thought about that a fair bit. But i kind thought that it read better going: created, started, run time, completed. It seemed to have a natural flow to it.

But TBH, i am not too bothered if you think it would be better next to the state.

**Comment by timotheeguerin** [comment]
Well I think as we have color here too that would work better

File: app/components/task/browse/preview/task-runtime.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import "app/styles/variables";
+
+bl-task-runtime {
+    // inherits .failure and .success from base.scss
+    color: map-get($primary, 400);
```
Comment (timotheeguerin) [inline app/components/task/browse/preview/task-runtime.scss]: change to `$primary-color`

**Pull request merged**